pkg/apis/ocs/v1/storagecluster_types.go

```diff
@@ -155,6 +156,16 @@ type EncryptionSpec struct {
 	Enable bool `json:"enable,omitempty"`
 }
 
+// ArbiterSpec defines the arbiter options for the StorageCluster. If defined,
+// the Storage Cluster is deployed in stretch configuration. In stretch
+// configuration, a minimum of 2 OSD pods and 2 mon pods are created in each of
+// the 2 data zones. In the third zone(arbiter), only a mon pod is created.
+// Arbiter mon pods don't require the nodes to have the ocs node label.
+type ArbiterSpec struct {
+	// +optional
+	FailureDomainValue string `json:"failureDomainValue,omitempty"`
```

Comment:
@jarrpa @umangachapagain @obnoxxx @iamniting Although I still need to work on the placement code for arbiter, I would like to get your opinion on this change in the API.

Rook needs both the failure domain name and the 3 values for that failure domain to configure the stretch cluster. ocs-operator determines the failure domain name internally and therefore requires only the value of the failure domain that will be arbiter.

Is `FailureDomainValue` an acceptable name ? If not, any suggestions?

Reply:
Here is the change in Rook https://github.com/rook/rook/pull/6489/commits/b91f4211c9833f89b107599cd6fd091f4c7f06f2#diff-23d4ca6ed288cf0c307912d860846319f62eb5f60846961916c0395a82d76cfaR123

Reply:
1. How about calling it `failureDomainLabel`, the same as in Rook?
2. Seems like this property should be passed by the UI, depending on the failure domains supported by the UI. No need to rely on the internal computation of failure domain in OCS in arbiter clusters IMO.

Reply:
If I am not wrong, ocs-operator has been determining the failure domain itself. The UI too, for the arbiter config is going to pass only the value of the failure domain(for example us-east-2b).

IIUC,  in the UI, the failure domain is always shown as a zone.

Reply:
Rook expects the full label to be set in the `failureDomainLabel`, such as `topology.kubernetes.io/zone`, not just the failure domain. Is this not the same setting that OCS will pass on to rook? Or is this setting just intended to be `zone` (in this example)?

Reply:
Based on @jarrpa's and @travisn's comments, the name of the field is now ArbiterLocation, this is the input given to the ocs-operator in the StorageCluster CR and is just the value of the failure domain that is for arbiter. The failure domain label that is passed to rook is a complete label.

Reply:
> Based on @jarrpa's and @travisn's comments, the name of the field is now ArbiterLocation
>
Sounds good, but where is this comment?